### PR TITLE
Change for clarity

### PR DESCRIPTION
**Pull request by mikefay**
As per Chris Gagne's request, adding a little clarity around Keychain Access as an app, and P12 as the output file, as well as style guide changes and a means to reference API responses.

### CODE REVIEW

**Comment by aengusbates** [comment]
We also have a PDOC ticket related to this doc: https://jira.beanstream.com/browse/PDOC-414

No need to delay this code review on resolving the ticket.

- We need to describe applepay or include or link to an external doc that does. it does any include (or link out to) core information about applepay in canada (e.g. supported cards) 
- We don't have an "apple pay api"
- Highlight that we have a working demo app:  "You can find more about Bambora and Apple Pay on Github" - is not clear.
- "register a Merchant ID and download a Merchant Certificate (P12)" - you're not downloading a .p12 file from Apple - the .p12 file is exported from keychain at a later point
- Let's not duplicate api spec. "API Requests" section duplicates swagger doc, but with less detail 
- Code samples are an appendix - we should review how these are included -they should be framed in context of a flow. eg "do this", then a code sample, "do that", then another code sample

File: source/docs/guides/apple_pay/index.md [hunk 1]
```diff
@@ -85,9 +85,11 @@ When you make an `apple_pay` request to our Payments API, it'll be formatted in
 | payment_token | The encrypted Apple Pay token containing card holder details, generated from within the iOS app. |
 | complete | The type of transaction being performed. True indicates a Purchase, and false is a Pre-Authorisation. |
 
-## Additional Examples
+## API responses
```
Comment (aengusbates) [inline source/docs/guides/apple_pay/index.md]: "API responses" and "Additional examples" headers are both H2s - looks line one should be lower priority or the "API responses" section will be empty.

File: source/docs/guides/apple_pay/index.md [hunk 2]
```diff
@@ -30,9 +30,9 @@ You can find more about Bambora and Apple Pay on [Github](https://github.com/bam
 
 Before you accept Apple Pay with Bambora, you need to [register a Merchant ID and download a Merchant Certificate (P12)](https://developer.apple.com/library/content/ApplePay_Guide/Configuration.html) with Apple.
```
Comment (aengusbates) [inline source/docs/guides/apple_pay/index.md]: The certificate you download is not in .p12 format yet.

File: source/docs/guides/apple_pay/index.md (hunk 1 above)
Comment (mikefay) [inline source/docs/guides/apple_pay/index.md]: Not sure where the Responses bit came from. It's not even in my draft that I copy and pasted from.

Responses has been deleted.

File: source/docs/guides/apple_pay/index.md (hunk 2 above)
Comment (mikefay) [inline source/docs/guides/apple_pay/index.md]: I've made the change.

**Comment by aengusbates** [comment]
Closing pull for clarity. Will revisit pull when we review doc.

**Pull request closed without merging**